src/parser/content-evaluator-module.ts

```diff
@@ -119,11 +120,14 @@ export class ContentEvaluatorModule extends BaseModule {
         currentRelevance = relevancesByAi[currentComment.id];
       }
 
-      const currentReward = this._getRewardForComment(currentComment, currentRelevance);
+      const currentReward = this._getRewardForComment(currentComment, currentRelevance)
+        .mul(currentComment.score?.priority ?? 1)
+        .toDecimalPlaces(3);
```

Comment:
Why cutting to 3 decimals?

Reply:
The permit rewards have payouts upto 3 decimals, decimal js multiplication doesn't round it up to 3 decimals that's why and I don't get exactly what you mean by precision being lost as 0.1+0.2 always has that weird decimal
https://stackoverflow.com/questions/50778431/why-does-0-1-0-2-return-unpredictable-float-results-in-javascript-while-0-2

Reply:
Yes that is why we use DecimalJs so `0.1 + 0.2` will yield `0.3`. If any rounding happing it should be solely for display or at the end of the computations.

Reply:
ohhhh my bad